gdk4-wayland/src/lib.rs

```diff
@@ -1,5 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+// rustdoc-stripper-ignore-next
+//! # Rust GDK 4 Wayland bindings
+//!
+//! This library contains safe Rust bindings for [GDK 4 Wayland](https://docs.gtk.org/gdk4-wayland/) which
+//! is the Wayland GDK backend specific functions.
```

Comment:
```suggestion
//! This library contains safe Rust bindings for [GDK 4 Wayland](https://docs.gtk.org/gdk4-wayland/).
//! GDK is an intermediate layer that isolates GTK from the details of the windowing system.
//! GDK Wayland contains functions specific to the Wayland backend.
```

Reply:
Any reason for duplicating the GDK definition here?

Reply:
It is a separate file after all.